gnucash/gnome-utils/gnc-main-window.cpp

```diff
@@ -255,12 +270,19 @@ typedef struct GncMainWindowPrivate
     GHashTable *merged_actions_table;
     /** Set when restoring plugin pages */
     gboolean restoring_pages;
+
+    const gchar   *previous_plugin_page_name; //FIXMEb added
+    const gchar   *previous_menu_quailifier; //FIXMEb added
```

Comment:
There's a typo in the variable name. Better fix it before merging... (quailifier->qualifier).

Reply:
OK, will be in next push.